File: Javascript/Ejercicio/Ejercicio7.js

    const squareSurface=(base,height)=>{
        const superface = base * height / 2;
          process.stdout.write(`\n THE SURFACE OF SQUARE IS = ${superface}\n`);
    }

    let b2;
    
    process.stdout.write("please enter the base of the triangle \n");
    process.stdin.on('data',async function(data1) {
      const base = await parseFloat(data1.toString());
      if(typeof b2 == 'undefined' ){
        console.log("please enter the height of the triangle")
        b2 =1;
      };
      process.stdin.on('data',async function(data2) {
      const height = await parseFloat(data2.toString());
      squareSurface(base,height);
      });
  });
  
  
  

File: Java/Ejercicio/src/Ejercicio7.java
package Ejercicio.src;

import java.util.Scanner;

public class Ejercicio7 {
    public static void main(String[] args) throws Exception {
        Scanner keyboard = new Scanner(System.in);
        System.out.println("please enter the base of the triangle");
        double base = keyboard.nextDouble();
        System.out.println("please enter the height of the triangle");
        double height = keyboard.nextDouble();

        double superface = base * height / 2;

        System.out.println("\n THE SURFACE OF SQUARE IS = " + superface + "\n`");

        keyboard.close();
    }
}
